src/texts.js

```diff
@@ -0,0 +1,22 @@
+//TODO Move them directly in the DB?
```

Comment:
Hmm I'm not sure what would be the benefit. If they are in a file we can make them translatable easily in the future. There is no need for these to be dynamic, thus imho they shouldn't be in the db. People can change them via PR's.